portal/src/pages/search.vue

```diff
@@ -98,23 +72,23 @@
 
 <script>
 import { mapGetters } from 'vuex'
-import PageMixin from '~/pages/page-mixin'
-import Search from '~/components/Search'
 import FilterCategories from '~/components/FilterCategories'
 import Materials from '~/components/Materials'
 import Spinner from '~/components/Spinner'
 import {
-  generateSearchMaterialsQuery,
-  parseSearchMaterialsQuery,
-  addFilter,
+  addFilter, generateSearchMaterialsQuery,
+  parseSearchMaterialsQuery
 } from '~/components/_helpers'
+import PageMixin from '~/pages/page-mixin'
+import Search from '../components/Search'
```

Comment:
~ will always reference the source folder so regardless of where this particular component/page is located. Slightly easier than using relative paths.